tests/search/test-class-search.php

```diff
@@ -862,4 +862,61 @@ public function test__vip_search_filter__ep_allow_post_content_filtered_index()
 
 		$this->assertFalse( $enabled );
 	}
+
+	/*
+	 * Ensure the various options, constants, and rate limiting affect query integration in the right ways
+	 */
+	public function test__ep_skip_query_integration() {
```

Comment:
Minor: Should we split this off into individual tests? I personally find large tests like this hard to follow and they also tend to be fragile (if we change one aspect of how the query integration stuff works, the test may need to ba completely re-written).